tutorials/01-gaussian-mixture-model/01_gaussian-mixture-model.jmd

```diff
@@ -186,8 +242,168 @@ scatter(
 )
 ```
 
-```julia, echo=false, skip="notebook", tangle=false
+## Marginalizing Out The Assignments
+
+We can write out the marginal posterior of (continuous) $w, \mu$ by summing out the influence of our (discrete) assignments $z_i$ from 
+our likelihood:
+
+$$
+p(y \mid w, \mu ) = \sum_{k=1}^K w_k p_k(y \mid \mu_k)
+$$
+
+In our case, this gives us:
+
+$$
+p(y \mid w, \mu) = \sum_{k=1}^K w_k \cdot \operatorname{MvNormal}(y \mid \mu_k, I)
+$$
+
+
+### Marginalizing By Hand
+
+We can implement the above version of the Gaussian mixture model in Turing as follows:
+
+First, Turing uses log-probabilities, so the likelihood above must be converted into log-space:
+
+$$
+\log \left( p(y \mid w, \mu) \right) = \text{logsumexp} \left[\log (w_k) + \log(\operatorname{MvNormal}(y \mid \mu_k, I)) \right]
+$$
+
+Where we sum the components with `logsumexp` from the [`StatsFuns.jl` package](https://github.com/JuliaStats/StatsFuns.jl).
```

Comment:
One should not use StatsFuns but LogExpFunctions if one is only interested in `logsumexp`.